misk/src/main/kotlin/misk/web/HttpCall.kt

```diff
@@ -11,13 +11,31 @@ import okhttp3.RequestBody
 import okio.BufferedSink
 import okio.BufferedSource
 
+/**
+ * Information about the socket on which a HTTP call arrived.
+ */
+data class SourceSocket (
+  val type: Type = Type.NETWORK,
```

Comment:
Preferable to have the caller indicate the type of socket than assume a default. Calling code should have to think about the type of socket.